lib/manageiq/appliance_console/auth_utilities.rb

```diff
@@ -0,0 +1,87 @@
+require "uri"
+require "erb"
+
+module ManageIQ
+  module ApplianceConsole
+    module AuthUtilities
+      HTTPD_CONFIG_DIRECTORY = Pathname.new("/etc/httpd/conf.d")
+
+      def restart_httpd
```

Comment:
This functionality is also done here:
https://github.com/ManageIQ/manageiq-appliance_console/blob/master/lib/manageiq/appliance_console/external_httpd_authentication.rb#L81

That code could be updated to leverage this `AuthUtilities` method. Perhaps that could be done in a separate PR.

Reply:
it would be a different say message for the ipa cases,  but yes that too can call AuthUtilities. separate PR.

Reply:
I agree with @jvlcek .  This is duplicated code, and we should DRY it up (I think this PR, but I'm ok with a separate PR).  The message for IPA is barely different, so I don't think that's an issue.